recipe/bld.bat

```diff
@@ -1,47 +1,10 @@
 setlocal enabledelayedexpansion
```

Comment:
The full `bld.bat` for conda released in `defaults` is

```
setlocal enabledelayedexpansion
set CONDA_DEFAULT_ENV=
%PYTHON% conda.recipe\setup.py install
if errorlevel 1 exit 1

mkdir %PREFIX%\etc\fish\conf.d
if errorlevel 1 exit 1

copy %SRC_DIR%\shell\conda.fish %PREFIX%\etc\fish\conf.d
if errorlevel 1 exit 1
```

Probably best just to stick with that.